src/main/java/org/apache/accumulo/access/BytesWrapper.java

```diff
@@ -22,10 +22,12 @@
 import static java.util.Objects.checkFromIndexSize;
 import static java.util.Objects.checkIndex;
 
+import java.io.Serializable;
 import java.util.Arrays;
 
-final class BytesWrapper implements Comparable<BytesWrapper> {
+public final class BytesWrapper implements Comparable<BytesWrapper>, Serializable {
```

Comment:
If this will be in the public api then could just call it `Bytes`.  It would need a since tag.  Also could add a `byte[] toArray()` method that returns a copy of the array.  The `byteAt()` method should be made public.

Reply:
Backed out this change in e04689e. Renamed BytesWrapper to BytesImpl and created a Bytes interface.